client/my-sites/site-settings/form-analytics-stores.jsx

```diff
@@ -53,13 +69,52 @@ class FormAnalyticsStores extends Component {
 		return this.renderSettings( settings, disableAll );
 	};
 
+	renderEnhancedSettings = disableAll => {
+		const { translate } = this.props;
+		const settings = [
+			{
+				key: 'enh_ec_tracking',
+				label: translate( 'Enable Enhanced eCommerce' ),
+				explanation: translate(
+					'Before enabling, turn on enhanced eCommerce in your Google Analytics dashboard.'
```

Comment:
Can we link part of this (or something here) to https://support.google.com/analytics/answer/6032539?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/19888/commits/0701418362cd26b4dba6cf4c9a7edad07d387b95